av_imu_launch/config/av_gx5_15.yml

```diff
@@ -13,3 +19,550 @@ microstrain_inertial_driver:
+    device_setup : True
+
+    # Controls if the driver-defined settings are saved
+    #     false - Do not save the settings
+    #     true  - Save the settings in the device's non-volatile memory
+    save_settings : False
+
+    # Controls if the driver creates a raw binary file
+    #     false - Do not create the file
+    #     true  - Create the file
+    #
+    # Note: The filename will have the following format -
+    #       model_number "_" serial_number "_" datetime (year_month_day_hour_minute_sec) ".bin"
+    #           example: "3DM-GX5-45_6251.00001_20_12_01_01_01_01.bin"
+    # Note: This file is useful for getting support from the manufacturer
+    raw_file_enable : False
+
+    # (GQ7/CV7 only) Controls if the driver requests additional factory support data to be included in the raw data file
+    #     false - Do not request the additional data
+    #     true  - Request the additional channels (please see notes below!)
+    #
+    # Note: We recommend only enabling this feature when specifically requested by Microstrain.
+    #       Including this feature increases communication bandwidth requirements significantly...
+    #       for serial data connections please ensure the baudrate is sufficient for the added data channels.
+    raw_file_include_support_data : True
+
+    # The directory to store the raw data file
+    raw_file_directory : "/home/your_name"
+
+
+    # ******************************************************************
+    # Frame ID Settings
+    # ******************************************************************
+
+    # The mode in which we will publish transforms to the below frame IDs
+    #     0 - No transforms will be published between any of the non static frame ids. (if publish_mount_to_frame_id_transform is true, it will still be published, and so will the antenna and odometer transforms)
+    #     1 - Global mode:
+    #             Transform will be published from earth_frame_id to target_frame_id containing global position
+    #     2 - Relative mode:
+    #             Note: In order to use relative mode, you must configure filter_relative_position
+    #             Transform will be published from earth_frame_id to map_frame_id using relative position configuration
+    #             Transform between map_frame_id and target_frame_id will be published using position information reported by the device
+    # for more information, see: https://wiki.ros.org/microstrain_inertial_driver/transforms
+    tf_mode : 0
+
+    # # Frame ID that most header.frame_id fields will be populated with.
+    # frame_id: "imu_fsp_r"
+
+    # Frame IDs determining the transforms published by this node to aid in navigation. See https://www.ros.org/reps/rep-0105.html
+    # Note: If use_enu_frame is false, these frames (with the exception of earth_frame_id) will automatically have "_ned" appended to them.
+    mount_frame_id     : "base_link"  # Frame ID that the device is mounted on.
```

Comment:
Surely this should be the imu tf frame?

Reply:
I am not sure what is that for as there is a `frame_id` parameter at the begining of the file set to `imu_fsp_r`.

Checking the latest rosbags, I can see `imu_fsp_r` frame:

```
header:
  stamp:
    sec: 1715868613
    nanosec: 296254719
  frame_id: imu_fsp_r
orientation:
  x: -0.004922871297996596
  y: -0.014997674106881874
  z: 0.8132273922380694
  w: 0.5817319345052637
orientation_covariance:
- 0.01
- 0.0
- 0.0
- 0.0
- 0.01
- 0.0
- 0.0
- 0.0
- 0.01
```